aip/cloud/2602.md

```diff
@@ -0,0 +1,122 @@
+---
+aip:
+  id: 2602
+  scope: cloud
```

Comment:
I would prefer have a separate `cli` scope.

Reply:
I can understand that (and that's how I started this out), however this isn't "how all CLIs work at Google". It's how the GCP SDK's CLI works. It's specifically only for GCP (not even for GSuite). Any ideas on how we can reconcile that issue?

Reply:
Ah, I see. Hmm.
One option is to use `cloud-cli`, but that seems icky.

Alternatively, we can stick with what you have. I checked `.github/CODEOWNERS` and we could have a separate line for the 2600 block with a different approval team.